app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/DeployStatus.tsx

```diff
@@ -6,10 +6,10 @@ import {
   StageFailureMessage,
   IManifest,
 } from '@spinnaker/core';
-import { KubernetesManifestService } from '../../../../manifest/manifest.service';
```

Comment:
Also updated `tsconfig.json` so we don't have to have these relative paths everywhere.